com_zimbra_attachmentalert/attachmentalert.js

```diff
@@ -63,8 +63,8 @@ AttachAlertZimlet.prototype._initializeRegEx =
 function() {
 	if (this._attachWordsRegEx)
 		return;
-	this._attachStr = this.getMessage("AttachmentAlert_attach");
-	this._errorMsgStr = this.getMessage("AttachmentAlert_error_noattachment");
+	this._attachStr = (this.getMessage("AttachmentAlert_attach").indexOf('???')>=0 ? 'attach,bijlage' : this.getMessage("AttachmentAlert_attach"));
```

Comment:
In _dev folder this.getMessage will return strings like ???message_id???, I use additional if statements to see if I get ??? and then return the default english texts. If deployed from zip, 
the usual translation just works in Zimbra.